package.json

```diff
@@ -14,7 +14,7 @@
   "files": [
     "dist"
   ],
-  "module": "./dist/maplibre-gl-directions.es.js",
+  "module": "./dist/maplibre-gl-directions.js",
```

Comment:
Vite 3 generated `module.js` for ESM and `module.cjs` for CommonJS. In Vite 2 it was `module.es.js` and `module.js` respectively.